source/docs/essentials/installation.md

```diff
@@ -0,0 +1,170 @@
+our product.
+
+While it isn't
+[fully featured](https://github.com/front-commerce/front-commerce-lite#what-it-is-not),
+we believe that if you enjoy working with it, you will enjoy working with
+Front-Commerce since we use the same concepts and the same file structure. You
+could even reuse most of your code within Front-Commerce if you want to upgrade
+to the licensed version. Please visit
+[Front-Commerce Lite project page on Github](https://github.com/front-commerce/front-commerce-lite)
+to get started with it.
+
+</blockquote>
+
+## Installation using the skeleton
+
+**This is the recommended way to start a new Front-Commerce project.**
+
+Front-Commerce skeleton repository provides a quick way to get started with
+Front-Commerce with sane default configurations.
+
+### Get the code
+
+In order to install the skeleton, you need to follow these instructions:
+
+```bash
+git clone --depth=1 git@gitlab.com:front-commerce/front-commerce-skeleton.git
+cd front-commerce-skeleton
+npm install
+```
+
+It may take a while…
+
+### Configure the project
+
+This skeleton comes with default values for most of the configuration. Below are
+the configurations you will most likely have to change/create.
+
+#### Configure the instance
+
+Front-Commerce uses environment variables for runtime configurations depending
+on the instance that runs (local, staging, production).
+
+Copy the `.env.dist` sample file to a local `.env` and adapt configurations.
+
+#### Configure stores
+
+Configure the project by creating a `src/config/stores.js` file. It should be an
+object with stores codes as keys and configurations as values.
+
+Here is an example:
+
+```js
+module.exports = {
+  default: {
+    locale: "en-GB",
+    currency: "EUR",
+    default_country_id: "GB",
+    localeData: require("react-intl/locale-data/en"),
+    countries: IsoCountries =>
+      IsoCountries.registerLocale(require("i18n-iso-countries/langs/en.json"))
+  }
+};
+```
+
+<blockquote class="info">
+**Magento:** when using Front-Commerce with Magento, store codes
+must match Magento ones.
+</blockquote>
+
+#### Configure remote services
+
+In order for Front-Commerce to interact with remote headless services, you must
+ensure it is configured properly.
+
+For instance, to interact with Magento2 you will need to configure a few things:
+
+- `src/config/serviceKeys/magento2.js`: Magento2 keys (available in the
+  `System > Extensions > Integrations` page of Magento admin area)
+- `src/config/serviceKeys/cache.js`: you will need to ensure that the key
+  configured here is identical to Magento’s `fc_cache_api_token` custom variable
+  to enable cache invalidation from Magento2
+
+We invite you to dig into the other configurations and tweak them if you wish.
+See [our documentation](#TODO) for further information.
+
+#### Optionally configure the caching strategy
+
+To use caching locally, you can configure it in `src/config/caching.js`. For now
```

Comment:
Maybe I would say what is this caching about. Maybe saying that it's for improving the performance when fetching data from remote services.